handlers/age.go

```diff
@@ -22,15 +23,19 @@ func (f *Filter) UpdateAge(w http.ResponseWriter, req *http.Request) {
 	vars := mux.Vars(req)
 	filterID := vars["filterID"]
 
-	req = forwardFlorenceTokenIfRequired(req)
-
-	if err := f.FilterClient.RemoveDimension(req.Context(), filterID, "age"); err != nil {
+	collectionID := getCollectionIDFromContext(ctx)
+	userAccessToken, err := headers.GetUserAuthToken(req)
+	if err != nil{
```

Comment:
As discussed you can use `headers.IsNotFound(err)` which should be a bit clearer to anyone reading the code.